app/master/build_request.py

```diff
@@ -22,6 +22,14 @@ class BuildRequest(object):
         "job_name": "clusterrunner_configured_job",
```

Comment:
Ah, true. I suppose ClusterRunner could just set the job_name value to some constant like "_adhoc_" if config was specified without a job name.